server/controllers/request.js

```diff
@@ -0,0 +1,99 @@
+const User = require("../models/User");
+const Profile = require("../models/Profile");
+const Request = require("../models/Request");
+
+exports.createRequest = async (req, res, next) => {
```

Comment:
Since we're not using the `asyncHandler` here, let's wrap our async operations with a try/catch block 

Ditto where applicable